### ISSUE TITLE
Error while fetching recent builds

### ISSUE THREAD

**Issue by forrestkouakou** [fictitiouswizard/browserstack_api#1]
```
  File "/root/src/asterix/venv/lib/python3.8/site-packages/bsapi/app_automate/appium/builds.py", line 39, in recent_builds
    builds = [
  File "/root/src/asterix/venv/lib/python3.8/site-packages/bsapi/app_automate/appium/builds.py", line 40, in <listcomp>
    Build(**b["automation_build"])
TypeError: __init__() got an unexpected keyword argument 'build_tag'
```
I'm facing this issue since a couple of weeks now, I think BS updated their API and are now sending build_tag which is definitely not expected by the build model (bsapi.models.Build)

**Comment by fictitiouswizard** [comment]
Looks like tags may have been changed to build_tag.  I need to look at it for a minute.

**Comment by fictitiouswizard** [comment]
Should be fixed in 0.1.6.  Let me know if it is still not working for you.

**Comment by forrestkouakou** [comment]
> Should be fixed in 0.1.6. Let me know if it is still not working for you.

Awesome, thanks dude
